StripePayments/StripePayments/Source/API Bindings/Models/STPConnectAccountCompanyParams.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 public class STPConnectAccountCompanyParams: NSObject {
 
     /// The company’s primary address.
-    @objc public var address: STPConnectAccountAddress!
+    @objc public var address: STPConnectAccountAddress?
```

Comment:
Seems like a breaking change?

Reply:
Simply writing to `address` won't break (e.g. `connectAccountCompanyParams.address = ...` will still compile), but other usages could break. I think it's okay to ship without bumping the major version since so few users will be affected and Xcode gives helpful fix-its any users that do get broken by this change. 

I can update the changelog w/ this change though.

Reply:
sorry my comment didn't post from last week... +1 to updating the changelog.